src/home/WalletHome.tsx

```diff
@@ -96,6 +98,28 @@ function WalletHome({ route }: Props) {
     )
   }
 
+  // Scroll Aware Header
+  const scrollPosition = useSharedValue(0)
+  const [titleHeight, setTitleHeight] = useState(0)
+
+  const handleMeasureTitleHeight = (event: LayoutChangeEvent) => {
+    setTitleHeight(event.nativeEvent.layout.height)
+  }
+
+  const handleScroll = useAnimatedScrollHandler((event) => {
+    onScroll
```

Comment:
Is this statement doing anything? `onScroll` I believe is a function, so this seems like just a line that references the function and doesn't invoke it.
Also, I think you should be able to use a single event handler instead of the 2 different handlers based on the gate. I believe both are just extracting `event.contentOffset.y` and setting it into a variable. `onScroll` sets it on the `scrollPositionValue` (of type `Animated.Value`) and `handleScroll` sets it on `scrollPosition` (of type `SharedValue`). We could just make all consumers use the SharedValue or just have a single handler that sets both variables.

Reply:
I tried conditionally setting the scroll in fc03d3e4ceb26b0698b73bb54a63d3c5b7d279e9. It looks like the other scroll is only being used by the drawer header so cleaning this up might be easier once that is removed. Perhaps a ticket to clean-up the drawer header and add todo's linked to that?